src/audio/kpb.c

```diff
@@ -379,10 +379,9 @@ static int kpb_prepare(struct comp_dev *dev)
 
 	if (kpb->state == KPB_STATE_RESETTING ||
 	    kpb->state == KPB_STATE_RESET_FINISHING) {
-		trace_kpb_error_with_ids(dev, "kpb_prepare() error: "
-					 "can not prepare KPB "
-					 "due to ongoing reset, state log %d",
-					 kpb->state_log);
+		trace_kpb_error("kpb_prepare() error: can not prepare KPB "
```

Comment:
"cannot" as a single word, please

Reply:
@lyakh, why do you care? Both forms mean the same? :D

Reply:
Yeah, not worth waiting eternity for CI to finish just for that change

Reply:
@mrajwa I care because we write comments in English, and AFAIK "can not" is wrong here: https://www.grammarly.com/blog/cannot-or-can-not/